templates/test/infopanel.html.ep

```diff
@@ -81,7 +81,10 @@
                 % }
                 % if ($worker) {
                     <div>
-                        Assigned worker: <span title="VNC port: <%= 90 + $worker->instance %>"><%= $worker->to_string %></span>
+                        Assigned worker:
+                        <span title="VNC port: <%= 90 + $worker->instance %>">
+                            %= link_to $worker->to_string => url_for('admin_worker_show', worker_id => $worker->id)
```

Comment:
scratch my previous comment. The worker *does* show also for completed job. I don't know what happened for me in before.